From f0f692798a8f5544b43e3cb3205b1a0751b638a9 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Wed, 8 Dec 2021 15:48:19 +0100
Subject: [PATCH 0100/1695] plat-stm32mp1: conf: enable the WERROR check

Add by default the WERROR to check and block the build
when an error is detected.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 941f8b08978078afbdd05781a15a5396bd08fda3 plat-stm32mp1: conf: enable the WERROR check]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: I78fa1846a60b772d9fed24f69e010a7559a22c4a
---
 core/arch/arm/plat-stm32mp1/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 3ae03bb7d..7bcb38078 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -305,6 +305,7 @@ endif
 # Default enable some test facitilites
 CFG_ENABLE_EMBEDDED_TESTS ?= y
 CFG_WITH_STATS ?= y
+CFG_WERROR ?= y
 
 # Enable OTP update with BSEC driver
 CFG_STM32_BSEC_WRITE ?= y
-- 
2.47.2

